sympy/sets/tests/test_sets.py

```diff
@@ -349,13 +349,13 @@ def test_set_operations_nonsets():
     ]
     nums = [0, 1, 2, S(0), S(1), S(2)]
 
-    for s in sets:
-        for n in nums:
+    for si in sets:
```

Comment:
I don't understand how this is any different.

Reply:
Here flake8 complains that n (imported at top) is shadowed by a loop variable:
```
$ flake8 sympy/sets/tests/test_sets.py
sympy/sets/tests/test_sets.py:353:13: F402 import 'n' from line 15 shadowed by loop variable
```
I changed `n` to `ni`. I then thought it looked better if I also changed `s` to `si`.

Reply:
I was looking for `s`...I see what you mean.